packages/context/src/binding.ts

```diff
@@ -104,6 +109,62 @@ export enum BindingScope {
    * - req2.get('b1') ==> 0 (always)
    */
   SINGLETON = 'Singleton',
+
+  /*
```

Comment:
Had a chat with @raymondfeng regarding the new scopes. We probably want to mention that, when the specified scope is same as owner, it's equivalent as scope `SINGLETON`.

E.g.

```ts
    appCtx
      .bind('foo')
      .toDynamicValue(() => `foo.app.${generateUniqueId()}`)
      .inScope(BindingScope.APPLICATION);
```
is equivalent to 
```ts
    appCtx
      .bind('foo')
      .toDynamicValue(() => `foo.app.${generateUniqueId()}`)
      .inScope(BindingScope.SINGLETON);
```